internal/output/html/package_table_template.gohtml

```diff
@@ -55,19 +55,48 @@
   <tr>
     <td colspan="100%">
       <div id="table-tr-{{ $index }}-details" class="package-details">
-        {{ if and $element.LayerDetail (not (eq $element.LayerDetail.LayerID "")) }}
+        {{ if and $element.LayerDetail.LayerInfo.LayerMetadata (not (eq $element.LayerDetail.LayerInfo.LayerMetadata.Command "")) }}
+        {{ $index := $element.LayerDetail.LayerIndex }}
+        {{ $diffID := $element.LayerDetail.LayerInfo.LayerMetadata.DiffID }}
+        {{ $baseImageIndex := $element.LayerDetail.LayerInfo.LayerMetadata.BaseImageIndex }}
+        {{ $originalCommand := $element.LayerDetail.LayerInfo.LayerMetadata.Command }}
+        {{ $commandSet := formatLayerCommand $originalCommand }}
+        {{ $command := index $commandSet 0 }}
+        {{ $commandDetail := index $commandSet 1 }}
+        {{ $longCommand := false }}
+        {{ if gt (len $command) 109 }}
+        {{ $longCommand = true }}
+        {{ end }}
+        {{ $binaries := formatSlice $element.OSPackageNames }}
+        {{ if not (eq $binaries "") }}
+        <p><span class="package-detail-title">Installed binaries:</span> {{ $binaries }}</p>
+        {{ end }}
+        <div class="layer-command-container">
+          <p><span class="package-detail-title">Layer introduced in:&nbsp;</span></p>
+          <div class="tooltip">
+            <p># {{ $index }} layer</p>
```

Comment:
nit: the " layer" text seems a bit redundant. Also, I still think it's not much value to users to dedicate a whole row to a "# layer" text that gives a diff ID when hovered over. 

How about something like this instead?

![image](https://github.com/user-attachments/assets/246453d1-4125-4354-8011-03dff51fdea6)

Also changed the "In base image: True: image") to just "In base image: image" to make it a bit more concise.

Reply:
Updated